src/Mooc/Courses/Application/Create/CourseCreator.php

```diff
@@ -28,5 +35,20 @@ public function __invoke(CourseId $id, CourseName $name, CourseDuration $duratio
 
         $this->repository->save($course);
         $this->bus->publish(...$course->pullDomainEvents());
+        if($this->hasLogger())
+        {
+            $message = sprintf("Course named %s with id %s has been created", $name, $id);
+            $this->logger->log(LogLevel::INFO, $message);
```

Comment:
```suggestion
            $this->logger->info('Course created', ['name' => $name, 'id' => $id]);
```

Logging with this style, you'll be able to group logs and filter easily. Always use the context, variables in the log message are forbidden! :P